meshtastic/localonly.proto

```diff
@@ -111,10 +111,20 @@ message LocalModuleConfig {
    */
   ModuleConfig.NeighborInfoConfig neighbor_info = 11;
 
+  /*
+   * The part of the config that is specific to the Neighbor Info module
```

Comment:
This is the config specific to ambient lighting. CTRL+V repeat from the entry above it looks like.